locales/en/public.json

```diff
@@ -108,31 +108,31 @@
       }
     },
     "JFR_METRICS_CARD_DESCRIPTION": "Display common performance metrics from recent JFR data.",
-    "JFR_METRICS_CARD_DESCRIPTION_FULL": "Display a single performance metric from a list of supported JFR event types. Data is displayed from the present moment back to a specified duration. Note: these metrics are only available for a single target at a time. Second Cryostat-Web instances, for the same user or others, will not behave as expected when using this card type.",
+    "JFR_METRICS_CARD_DESCRIPTION_FULL": "Display a single performance metric from a list of supported JFR Event Types. Data is displayed from the present moment back to a specified duration. Note: these metrics are only available for a single target at a time. Second Cryostat-Web instances, for the same user or others, will not behave as expected when using this card type.",
     "JFR_METRICS_CARD_TITLE": "JFR Metrics Chart",
     "MBEAN_METRICS_CARD_DESCRIPTION": "Display common performance metrics from current MBean data.",
     "MBEAN_METRICS_CARD_DESCRIPTION_FULL": "Display a single performance metric from a list of supported MBeans.",
     "MBEAN_METRICS_CARD_TITLE": "MBean Metrics Chart",
     "NO_RECORDING": {
-      "DESCRIPTION": "Metrics cards display data taken from running flight recordings with the label <label>origin={{recordingName}}</label>. No such recordings are currently available.",
-      "TITLE": "No source recording"
+      "DESCRIPTION": "Metrics cards display data taken from running flight Recordings with the label <label>origin={{recordingName}}</label>. No such Recordings are currently available.",
```

Comment:
This probably should be `Flight Recordings`, since that comes from `(J)FR` as a specific name.